docs/UserGuide.md

```diff
@@ -115,46 +115,46 @@ section of this user guide or to an external website.
    ![Ui](images/GuestBookIcon.png) <br><br>
 
 4. Double-click on `guestbook.jar` to start the app. 
-   * Alternatively, if you wish to use the `Terminal` or `Command Prompt`, enter 
+   * Alternatively, if you wish to use the `Terminal` or `Command Prompt`, navigate to the **home folder** and enter 
```

Comment:
Should we include how to navigate?

Reply:
I think it is alright to not input as it really depends on the home folder, and directing the user to view navigating commands for Windows, Mac and Linux can be quite overwhelming to include in the User Guide. I think it can be safe to assume that if they prefer to use the Terminal or Command Prompt, they should be quite decent at navigating it as they would be rather tech-savvy.